### PR TITLE
Renamed: ABTestReponse -> ABTestResponse

### LINKED ISSUE #597: Bad naming in ABTestsReponse
- Algolia Client Version: latest

### Description

The object `ABTestsReponse` should be renamed to `ABTestsResponse`.

### PR DESCRIPTION
**Pull request by Ant-hem**
| Q                 | A
| ----------------- | ----------
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no     
| Related Issue     | Fix #597 
| Need Doc update   | yes/no

## Describe your change

Typo in the class name.

### CODE REVIEW

**Comment by Ant-hem** [comment]
> It's breaking IMO. What do you think?

It is but with a proper IDE it's a 1 sec change :-) If commented in the changelog it's not a huge BC break.

**Comment by nunomaduro** [comment]
@Ant-hem It's up to you buddy. To give you my context, on the PHP Deployments people specify on their deployment scripts : `algolia: ^2.0` - this means they will get `minors|patchs` in production. I don't know if this is the case in csharp.

File: src/Algolia.Search/Models/Analytics/ABTestsReponse.cs [hunk 1]
```diff
@@ -29,7 +29,7 @@ namespace Algolia.Search.Models.Analytics
     /// <summary>
     /// Api response
     /// </summary>
-    public class ABTestsReponse
+    public class ABTestsResponse
```
Comment (nunomaduro) [inline src/Algolia.Search/Models/Analytics/ABTestsReponse.cs]: The file also needs to be renamed.

**Comment by Ant-hem** [comment]
Usually in the .NET/C# world people put a fixed version in their `.csproj`. Even if they don't, the build will fail and they can fix the issue in staging. I'll keep this fix for a `6.X` release.

**Pull request merged**